Hookies/Game/System/StartSystem.swift

```diff
@@ -30,7 +30,7 @@ class StartSystem: System, StartSystemProtocol {
     }
 
     private var expectedNumberOfPlayers: Int {
-        return players.count
+        return players.filter({ $0.key.playerType == .human }).count
```

Comment:
players.count will include the bots, then since the bots cannot send out a game connection event, then the startSystem will keep waiting to start. 
Is it necessary for the bot to send out the game connection event in order for the game to work properly? (If it is necessary then how to handle offline ah?)